Modules/vc-module-catalog/VirtoCommerce.CatalogModule.Data/ExportImport2/CatalogFullExportPagedDataSource.cs

```diff
@@ -102,7 +102,7 @@ public CatalogFullExportDataQuery DataQuery
 
                 _exportDataSourceStates.Add(new ExportDataSourceState
                 {
-                    SearchCriteria = new ProductSearchCriteria() { ResponseGroup = (ItemResponseGroup.Full & ~ItemResponseGroup.Variations).ToString() },
+                    SearchCriteria = new ProductSearchCriteria() { CatalogIds = _dataQuery.CatalogIds, ResponseGroup = (ItemResponseGroup.Full & ~ItemResponseGroup.Variations).ToString() },
```

Comment:
Are variations included in search result? I think we need to set
```
SearchInVariations = true
```
for that.

Reply:
I'm not sure. This initialization just copied from old export.